cns/networkcontainers/networkcontainers_windows.go

```diff
@@ -40,7 +38,7 @@ func updateInterface(createNetworkContainerRequest cns.CreateNetworkContainerReq
 	return nil
 }
 
-func setWeakHostOnInterface(ipAddress string) error {
+func setWeakHostOnInterface(ipAddress, ncID string) error {
```

Comment:
is this function even needed now?

Reply:
I mean separately?

Reply:
Is there way to asset the lock has been acquired already?

Reply:
asset the lock?
Keeping the function separate because it's still a separate functionality